src/Components/Input/index.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { shallow } from "enzyme";
+import Input from "./index";
+
+describe("Input component", () => {
+  test("should render properly", () => {
+    const wrapper = shallow(<Input />);
```

Comment:
- [ ] For the `should render properly` tasks let's use `mount` instead of `shallow`

Reply:
Got it! :)